package.json

```diff
@@ -62,9 +62,9 @@
     }
   },
   "dependencies": {
+    "@ai-sdk/react": "1.2.10",
```

Comment:
Our version of `ai/react` was simply re-exporting `@ai-sdk/react` version 1.2.9 (This also updates to latest, which is 1.2.10). See https://github.com/vercel/ai/blob/e69fb206d8005b40ba8bc3a02898b55bccbebac8/packages/ai/react/index.ts#L1

**Explanation:** I'm not 100% sure why this resolves the type errors @jkachel encountered on https://github.com/mitodl/smoot-design/pull/29, but (A) it does, and (B) I've been noticing warnings that `ai/reat` is deprecated and we should use `@ai-sdk/react` instead anyway.